docs/runtime.md

```diff
@@ -122,9 +122,23 @@ Finally, if you want to change CRI look into:
 By default, CRI is set to runC. As such, you must configure Nvidia GPU support by replacing `runc` with `nvidia-container-runtime`:
 
 ```toml
-[plugins."io.containerd.runtime.v1.linux"]
-    shim = "containerd-shim"
-    runtime = "nvidia-container-runtime"
+[plugins."io.containerd.grpc.v1.cri".containerd.default_runtime]
```

Comment:
aint sure, why do we need to have here settings for default_runtime as well with the nvidia runtime settings?

Reply:
if you want to use `nvidia-container-runtime` as your default instead of switching between different ones. I didn't want to use `runc` on my GPU node. I don't know how to pass the info to `containerd` via Pod manifest to use `nvidia-container-runetime`